coderd/coderd.go

```diff
@@ -577,6 +601,13 @@ func New(options *Options) *API {
 		})
 	})
 
+	if options.SwaggerEndpointEnabled {
+		// Swagger UI requires the URL trailing slash. Otherwise, the browser tries to load /assets
+		// from http://localhost:8080/assets instead of http://localhost:8080/swagger/assets.
+		r.Get("/swagger", http.RedirectHandler("/swagger/", http.StatusSeeOther).ServeHTTP)
```

Comment:
Maybe should use `Temporary Redirect` instead. `See Other` is more for redirecting after a POST/PUT AFAIK

Reply:
Fixed